src/redux/books/books.js

```diff
@@ -0,0 +1,42 @@
+// action types
+const ADDED_BOOK = 'book added';
+const DELETED_BOOK = 'book deleted';
+
```

Comment:
- [ ] Hi. The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). 
 i.e  `const ADDED_BOOK = redux/books/ADDED_BOOK`(following your workflow)